pkgs/development/python-modules/google-auth/default.nix

```diff
@@ -48,6 +48,7 @@ buildPythonPackage rec {
     ];
     enterprise_cert = [
       cryptography
+    ] ++ lib.optionals (!stdenv.isDarwin || !stdenv.isAarch64) [
```

Comment:
```suggestion
```
This does not reflect upstream, so we are not doing to do it. https://github.com/googleapis/google-auth-library-python/blob/main/setup.py#L43 Rather we should conditionally remove enterprise_cert from checkInputs and disable the failing tests for M1 Macs only.

Reply:
Updated :+1: